dexcom/fetch/translate.go

```diff
@@ -388,8 +376,26 @@ func translateEGVToDatum(egv *dexcom.EGV, unit *string, rateUnit *string) data.D
 				"threshold": dexcom.EGVValuePinnedMgdLMaximum,
 			}}
 		}
+	case dexcom.EGVUnitMmolL:
+		if *datum.Value < dexcom.EGVValuePinnedMmolLMinimum {
+			datum.Value = pointer.FromFloat64(dexcom.EGVValuePinnedMmolLMinimum - 0.1) //check this
```

Comment:
More of a question for my understanding. If the value is lower than the theshold minimum, we want to still keep it below the minimum but only by a small amount?

Reply:
yes - I don't understand the reasoning 100% but that seems to be the decision